cert/selfsigned.go

```diff
@@ -272,11 +273,11 @@ func GenCertPair(org, certFile, keyFile string, tlsExtraIPs,
 	}
 
 	// Write cert and key files.
-	if err = ioutil.WriteFile(certFile, certBuf.Bytes(), 0644); err != nil {
+	if err = io.WriteFileTransactional(certFile, certBuf.Bytes(), 0644); err != nil {
```

Comment:
I believe there should have been an `os.Remove` here. I did not add a removal to any other spots.